tests/IntegrationTest.php

```diff
@@ -229,7 +229,7 @@ public function nl2br($value, $sep = '<br />')
     {
         // not secure if $value contains html tags (not only entities)
         // don't use
-        return str_replace("\n", "$sep\n", $value);
+        return str_replace("\n", "$sep\n", $value ?? '');
```

Comment:
Why is it needed?

Reply:
Because of this:
```

  1x: str_replace(): Passing null to parameter #3 ($subject) of type array|string is deprecated
    1x in IntegrationTest::testIntegration from Twig\Tests
```